### ISSUE TITLE
How can I modify the local CP config?

### ISSUE THREAD

**Issue by kevinreyn** [jlesage/docker-crashplan-pro#68]
Apparently there was/is a configuration setting that is not updated when you make changes to to the gui for deduplification.  I am trying to improve my bandwidth use in CP and I discovered a potential fix but have no idea how to make this change in the container version of CP.
In my.service.xml  there are these two lines. 

<dataDeDupAutoMaxFileSize>1073741824</dataDeDupAutoMaxFileSize> <dataDeDupAutoMaxFileSizeForWan>0</dataDeDupAutoMaxFileSizeForWan>

I would like to change the value in the forwan to something different than 0    or see if the client changes for dedup actually change the value or not.    In the past people have seen real bandwitch changes when upping that value. 

Thank you for your support on this container - its a solid solution

**Comment by jlesage** [comment]
The `my.service.xml` file can be found in your appdata folder (i.e. the folder on the host that you mapped to `/config`), under the `conf` directory.

But note that while you can see better upload speed, you will also upload more data.  Also, make sure to read the related article from CrashPlan:

https://support.code42.com/CrashPlan/4/Configuring/Unsupported_changes_to_CrashPlan_de-duplication_settings

**Comment by kevinreyn** [comment]
Yeah I read that article and I understand the ramifications.
Problem I encounter is my backup set is HUGE and 98.9% of it is large nef,
psd and psb files.
The backup set is an offsite archive of content.
What I encounter is it takes a very long time for it to evaluate every file
in the backupset for change when I know that the vast majority of files in
the set have not changed.
so setting that value to 30meg means non of the raws will ever be checked
to dedupe because they never change.  When I am working on the PSD and
PSB's they never even hit the nas until they are finished etc. etc
All the little files that do change often will still get picked up.  I
would not recommend every user do this but I know my data being archived
well and know this will actually speed my backup.

Consequently I will have to look at my config for my container because I
dont recall setting that value.

Regardless Code42 should pay you because had I not found your solution to
the nas implementation of the client in Docker I would have bailed on them
when they screwed all the Crashplan Home users

Thank you so much for your efforts your a life saver

**Comment by jlesage** [comment]
Great so let me know by re-opening this issue if you still need help on this.

**Comment by kevinreyn** [comment]
Hey I did have one other question - how do I control the memory use of the
container.  Mine gets really really big even though I have only given the
app 8g the container ends up around 14g

**Comment by jlesage** [comment]
How do you look at the memory usage of the container?  When you say that you give 8GB to the app, are you talking about the `CRASHPLAN_SRV_MAX_MEM` variable?

CrashPlan consumes a lot of memory.  To my knowledge, you can't really "control" or limit the amount memory it consumes.  Doing so will result in crashes and prevent the app the backup successfully your data.

According to 
https://support.code42.com/CrashPlan/6/Troubleshooting/Adjust_Code42_app_settings_for_memory_usage_with_large_backups, you need about 1GB of memory per TB of data.  The article also gives some tips on how to reduce the amount of required memory.

**Comment by kevinreyn** [comment]
Yes that’s the value.  That limits. Or more importantly expands the memory
space for college to build its backup map.  Mine currently consumes about
4.5g of that 8.  I ran into the backup set memory issue several years ago.

What I am more curious about is how to co trip the amount of memory that
the container uses.  Right now it ramps up to 12-14 gig on my 16 gig nas.
So trying to figure out how to manage that better.   CP running native on
the syn usually only consumed a little over what I had allocated to it.

Here’s the weird part.  Docker reports the container using as much as 14g
and turns red.  But if I look at system information overall memory
allocation does not match what docker is saying
